src/en/space-station-14/round-flow/proposals/revolutionaries-codeword-rework.md

```diff
@@ -0,0 +1,259 @@
+
+Codewords come in three levels; Low, Mid and High Conversion words, with **4** codewords given in each level.
+  - Low Conversion words are meant to be easy to incorporate into normal speech. There should be so many possible low conversion words that can be rolled that it becomes effectively impossible to metagame. Examples: "love", "hello", "great", "hate".
+  - Mid Conversion words deal a decent amount of CHP damage but can be noticed if one pays attention. These would be at the risk of metagaming due to being flavored towards revolutionary speech, so it would be on the rev player to incorporate them well into conversations. Examples: "disrupt", "rise up", "join".
+  - High Conversion words are very clearly revolutionary and deal high CHP damage, *and is able to convert a crewmember in the Convertable state. They should be mostly useful when the revolution is in full swing, but risky for a non-convertable crewmember to overhear. Examples: "viva", "communism", "comrade".
+
+Dealing damage with codeword has a **10** second cooldown, to avoid simply spamming the codewords. Damage is also split among the crew who hear it, making speech directed to a single individual more effective at targeted conversion, while multiple Revolutionaries can shout it out to crowds without the conversion happening to fast.
+
+Codewords are proposed to deal the following damage:
+  - Low: **10** CHP
+  - Mid: **20** CHP
+  - High: **30** CHP, and converts any crew in the Convertable state.
+
+Codewords are able to deal damage even when communicated across different mediums, however the conversion strength is reduced to compensate for how easy it may be to do so:
+
+- A Revolutionary speaking a codeword through a headset radio deals **0.25**x CHP damage.
+- Announcement messages with a codeword by a Revolutionary deals **0.5**x CHP damage, without being divided among the people hearing it.
+
+### Deconversion
+
+Deconversion is the main way for crew to fight back against the revolution. It should be fairly easy when targeted at isolated Revolutionaries but require more work when Revolutionaries are together. It should also not happen by accident; crew should consciously attempt it to be successful.
+
+Deconversion puts you fully on the crew side. This does fully include the ability to sell out your fellow Revolutionaries by remembering their names. Headrevs therefore have a choice to act more stealthily disguise themselves and blend in with the Revolution, or assign another Revolutionary (possibly their first convert) to act as the revolution's leader to throw off potential deconvert sell-outs. 
+
+Methods of deconversion include:
+  - Harm baton, heals **10** CHP per hit
```

Comment:
I feel like this makes a crew lose before the round ever reaches the open confrontation, arresting a single antag in a couple minutes is considered pretty fast normally, but would be deadly here. Might be fine with some number wrangling, but wanted to give an alternative:

I love the idea to nudge sec to be violent about this, possibly lethal after a certain point, but some nonviolent ways should be available to the rest of the crew. Perhaps being around the captain humanizes him so he passively heals CHP. Or alcohol/space drugs makes people content so it greatly increases the healing rate. The random pizza event deals global CHP damage, but actually eating it heals a ton. Stuff like this would actively push revs to be increasingly more public to counteract these effects. It could also make paranoid heads do the things corpo managers do to keep morale up, which could be pretty funny.

I agree that deconversion should not happen on accident and it makes these tricky, but they could work similarly to how poison works in a lot of games. Simply can't go over 0, and/or the effect scales in a way that can't deconvert, but the heal amount is greater the further away from 0 the person is.